general_rules/PenaltiesBonuses.tex

```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 	\item It is the decision of the \iaterm{Technical Committee}{TC} if (and to which degree) the bonus score is granted.
 \end{enumerate}
 
+\subsection{Bonus for perceived social intelligence}\label{rule:perceived_intelligence}
+\begin{enumerate}	
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
```

Comment:
Imho should be mandatory and integrated in the scoring of the tasks - to make sure league/referee prepares volunteers.
To be useable data we have to define a minimum number of respondents, 
for testing the procedure (or getting an overview of the league) in local leagues we can always opt to skip parts of a task e.g. this survey.

Reply:
I don't think it has to be mandatory, but I agree, that it should be integrated in the scoring of the tasks. Since you can't lose anything anyway, any rational team should always opt for social scoring.

Reply:
Yes, I think we can just make it mandatory

Reply:
~~Teams are allowed~~ Teams are assessed